ext/Pod-Html/lib/Pod/Html.pm

```diff
@@ -543,6 +530,7 @@ sub parse_command_line {
                        'recurse!'   => \$opt_recurse,
                        'title=s'    => \$opt_title,
                        'verbose!'   => \$opt_verbose,
+                       'xhtml'      => \$opt_xhtml,
```

Comment:
I think it more prudent to keep the current mode and turn on HTML 5 by having an `--html5` mode.

My concern is what would break for various other non-HTML5 browsers. Before promoting making HTML 5 the default, I would look at `lynx`, `elinks`, et. al. to understand how they deal with it.

Reply:
I just installed and tested with `lynx 2.8.9` and `elinks 0.12-0.62` and both render the HTML5 how I would expect a text browser to do so. Both are 100% readable and happy.